src/watchFiles.ts

```diff
@@ -1,14 +1,16 @@
-require('dotenv').config();
-const fs = require('fs');
-const path = require('path');
-const chokidar = require('chokidar');
-const chalk = require('chalk');
+import dotenv from 'dotenv';
+import fs from 'fs-extra';
+import path from 'path';
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+
+dotenv.config();
 
 const folderName = 'Downloads';
-const basePath = path.resolve(process.env.HOMEDRIVE, process.env.HOMEPATH);
-const folderToWatch = path.resolve(basePath, folderName);
+const basePath: string = path.resolve(process.env.HOMEDRIVE!, process.env.HOMEPATH!);
+const folderToWatch: string = path.resolve(basePath, folderName);
```

Comment:
You probably don't need to be super specific here, as long as `path.resolve` always returns a stirng then let TypeScript type this automatically for you.